postgres/sql/meshinfo.sql

```diff
@@ -0,0 +1,93 @@
+CREATE TABLE nodes (
+  id bigint NOT NULL PRIMARY KEY,
+  first_heard_at timestamp with time zone DEFAULT now() NOT NULL,
+  last_heard_at timestamp with time zone DEFAULT now() NOT NULL
+);
+
+CREATE TABLE node_infos (
+  id bigint NOT NULL PRIMARY KEY,
+  node_id bigint NOT NULL,
+  long_name character varying(100),
+  short_name character varying(10),
+  mac_addr character varying(20),
+  hw_model character varying(20),
+  role character varying,
+  created_at timestamp with time zone DEFAULT now() NOT NULL
+);
+
+CREATE TABLE node_positions (
+  id bigint NOT NULL PRIMARY KEY,
+  node_id bigint NOT NULL,
+  latitude double precision NOT NULL,
+  longitude double precision NOT NULL,
+  altitude double precision,
+  geom public.geometry(PointZ,4326),
```

Comment:
If we're going to leave in the 'geom' columns, we're going to need to ensure the postgis extensions are loaded, either ad DB setup or at some point later.

Reply:
Good point, let me add this to the top. Any thoughts on other extensions we are missing?